fastapi/exceptions.py

```diff
@@ -1,20 +1,8 @@
-from typing import Any, Dict, Optional, Sequence, Type
+from typing import Any, Sequence, Type
 
 from pydantic import BaseModel, ValidationError, create_model
 from pydantic.error_wrappers import ErrorList
-from starlette.exceptions import HTTPException as StarletteHTTPException
-
-
-class HTTPException(StarletteHTTPException):
-    def __init__(
-        self,
-        status_code: int,
-        detail: Any = None,
-        headers: Optional[Dict[str, Any]] = None,
-    ) -> None:
-        super().__init__(status_code=status_code, detail=detail)
-        self.headers = headers
-
+from starlette.exceptions import HTTPException as HTTPException  # noqa
```

Comment:
Is the renaming of import, i.e., `as HTTPException` needed at the last?

Reply:
```suggestion
from starlette.exceptions import HTTPException
```

What's the reason for `noqa`?

Reply:
@tiangolo, why leave it dangling?